Aidan/multiply.py

```diff
@@ -0,0 +1,22 @@
+#! C:\Users\HP\local repository\AWS\Aidan
```

Comment:
Let's change this line to make the file executable.

Reply:
Copy that